miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/permission/ui/MiniAppCustomPermissionWindow.kt

```diff
@@ -107,8 +108,19 @@ internal class MiniAppCustomPermissionWindow(
 
         customPermissionLayout.findViewById<TextView>(R.id.permissionCloseWindow)
             .setOnClickListener {
-                customPermissionBridgeDispatcher.sendCachedCustomPermissions()
-                customPermissionAlertDialog.dismiss()
+                onNoPermissionsSaved()
             }
+
+        customPermissionAlertDialog.setOnKeyListener { _, keyCode, event ->
+            if (keyCode == KeyEvent.KEYCODE_BACK && event.action == KeyEvent.ACTION_DOWN) {
```

Comment:
It seems a little strange to trigger this on `ACTION_DOWN`. Usually I expect a button to trigger only after I've released it, so on `ACTION_UP`.  What do you think @minh-rakuten @khairul-alam-licon ?

Reply:
Yes, `KeyEvent.ACTION_UP` makes more sense in this usecase. I have updated this part.